pom.xml

```diff
@@ -8,62 +8,88 @@
   <artifactId>plantbattles</artifactId>
   <version>1</version>
   <packaging>war</packaging>
- 
-  <properties>
+
+ <properties>
+
     <!-- This project uses Java 8 -->
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <failOnMissingWebXml>false</failOnMissingWebXml>
-  </properties>
- 
+</properties>
+
   <dependencies>
     <dependency>
-      <groupId>javax.servlet</groupId>
-      <artifactId>javax.servlet-api</artifactId>
-      <version>4.0.1</version>
-      <scope>provided</scope>
+        <groupId>javax.servlet</groupId>
+        <artifactId>javax.servlet-api</artifactId>
+        <version>4.0.1</version>
+        <scope>provided</scope>
```

Comment:
I hate to say it lol but there are extra white spaces here

Reply:
pain